app/src/App.tsx

```diff
@@ -23,33 +29,44 @@ import PaperTraderApi from './Api'
  */
 
 function App() {
+	const [token, setToken] = useState(localStorage.getItem('userToken'));
+	const [currentUser, setCurrentUser] = useState<UserData | null>(null);
```

Comment:
```suggestion
	const [currentUser, setCurrentUser] = useState<UserData>(null);
```
this is the same as the above

Reply:
Since the initial state is `null`, need to use `| null` or else there will be a type assignment error. Similar to this [discussion](https://stackoverflow.com/a/59126275)

Reply:
https://stackoverflow.com/a/61823158/7995306
it's already implied by the unitialized state without specifying in the `<>`

also, `undefined` may be preferred here instead of `null`

Reply:
Made the change to use undefined instead of null